packages/core/src/table/table.stories.ts

```diff
@@ -0,0 +1,491 @@
+          <td>Solid maybe</td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <th>Footer</th>
+          <th>Footer</th>
+          <th>Footer</th>
+          <th>Footer</th>
+        </tr>
+      </tfoot>
+    </table>
+  `;
+}
+
+/** @website */
+export function leftTable() {
+  return html`
+    <table cds-table="left">
+      <caption>
+        Demo with the cells left aligned in a Clarity table.
+      </caption>
+      <thead>
+        <tr>
+          <th>Wizard</th>
+          <th>Allegiance</th>
+          <th>Triwizard Champion?</th>
+          <th>Can Cast Fireball</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Harry</td>
+          <td>Gryffindor</td>
+          <td>Yes</td>
+          <td>No</td>
+        </tr>
+        <tr>
+          <td>Gandalf</td>
+          <td>Hobbits</td>
+          <td>Maybe?</td>
+          <td>I don&apos;t think so...</td>
+        </tr>
+        <tr>
+          <td>Obi-Wan Kenobi</td>
+          <td>Republic/Rebellion</td>
+          <td>No</td>
+          <td>No</td>
+        </tr>
+        <tr>
+          <td>Merlin</td>
+          <td>King Arthur</td>
+          <td>Probably invented the tournament</td>
+          <td>Solid maybe</td>
+        </tr>
+      </tbody>
+    </table>
+  `;
+}
+
+/** @website */
+export function rightTable() {
+  return html`
+    <table cds-table="right">
+      <caption>
+        Demo with the cells right aligned in a Clarity table.
+      </caption>
+      <thead>
+        <tr>
+          <th>Wizard</th>
+          <th>Allegiance</th>
+          <th>Triwizard Champion?</th>
+          <th>Can Cast Fireball</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Harry</td>
+          <td>Gryffindor</td>
+          <td>Yes</td>
+          <td>No</td>
+        </tr>
+        <tr>
+          <td>Gandalf</td>
+          <td>Hobbits</td>
+          <td>Maybe?</td>
+          <td>I don&apos;t think so...</td>
+        </tr>
+        <tr>
+          <td>Obi-Wan Kenobi</td>
+          <td>Republic/Rebellion</td>
+          <td>No</td>
+          <td>No</td>
+        </tr>
+        <tr>
+          <td>Merlin</td>
+          <td>King Arthur</td>
+          <td>Probably invented the tournament</td>
+          <td>Solid maybe</td>
+        </tr>
+      </tbody>
+    </table>
+  `;
+}
+
+/** @website */
+export function modifierTable() {
+  return html`
+    <table cds-table>
+      <caption>
+        Demo with the cells right aligned in a Clarity table.
+      </caption>
+      <thead>
+        <tr>
+          <th>Wizard</th>
+          <th>Allegiance</th>
+          <th>Triwizard Champion?</th>
+          <th>Can Cast Fireball</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td cds-col="left">Use cds-col="left" to left align cell</td>
+          <td>Gryffindor</td>
+          <td>Yes</td>
+          <td cds-col="right">Use cds-col="right" to right align cell</td>
```

Comment:
I was going to ask about this. 🏆

Reply:
Was not sure what other modifiers we might support though?